src/commands/Admin/roleset.ts

```diff
@@ -63,11 +63,11 @@ export default class extends SkyraCommand {
 		});
 
 		return message.send(
-			language.get(created ? LanguageKeys.Commands.Admin.RoleSetCreated : LanguageKeys.Commands.Admin.RoleSetAdded, {
+			t(created ? LanguageKeys.Commands.Admin.RoleSetCreated : LanguageKeys.Commands.Admin.RoleSetAdded, {
 				name,
-				roles: language.list(
+				roles: t(
 					roleSets.map((role) => role.name),
-					language.get(LanguageKeys.Globals.And)
+					t(LanguageKeys.Globals.And)
```

Comment:
Isn't this.. wrong?

Reply:
Yeah, it should use the andList formatter.